File: add/codesnippet/VisualBasic/p-system.web.configurati_130_1.vb
                    ' Get the urlMappings section.
                    Dim urlMappings _
                    As UrlMappingsSection = _
                    systemWeb.UrlMappings
                    ' Read section information.
                    info = urlMappings.SectionInformation

                    name = info.SectionName
                    type = info.Type
                    declared = info.IsDeclared.ToString()
                    msg = String.Format("Name:     {0}" + _
                    ControlChars.Lf + "Declared: {1}" + _
                    ControlChars.Lf + "Type:     {2}" + _
                    ControlChars.Lf, name, declared, type)

File: add/codesnippet/CSharp/p-system.web.configurati_130_1.cs
                        // Get the urlMappings section.
                        UrlMappingsSection urlMappings =
                            systemWeb.UrlMappings;
                        // Read section information.
                        info =
                            urlMappings.SectionInformation;

                        name = info.SectionName;
                        type = info.Type;
                        declared = info.IsDeclared.ToString();
                        msg = String.Format(
                            "Name:     {0}\nDeclared: {1}\nType:     {2}\n",
                            name, declared, type);